src/server/models/userModel.ts

```diff
@@ -11,21 +12,21 @@ interface LoginDetails {
 }
 
 class UserModel {
-  private cachedLogins: {[id: string]: LoginDetails};
+  async getLoginDetails(login: string): Promise<LoginDetails|null> {
+    const tokensCollection = await firestore()
```

Comment:
Unclear what kind of tokens these are. Can you add "github" to the naming in the db and vars here?